### DISCUSSION TITLE
How to use Web Cam + yolov5+sahi

### DISCUSSION THREAD

**Discussion by VYRION-Ai** [obss/sahi#648] [Q&A] [answered]
How to use Web Cam + yolov5+Sahi

**Comment by fcakyon** [comment]
Hello @totoadel ! `sahi` currently dont have webcam support included but we are working on video support: https://github.com/obss/sahi/pull/442

**Comment by VYRION-Ai** [comment]
@fcakyon why?
it work with loop of images, what is the different between loop of images an frames from webcam

**Comment by fcakyon** [comment, marked as answer]
I mean `sahi` can support it but the implementation is not present at the moment. Feel free to open a PR for the camera feed support 👍

**Comment by VYRION-Ai** [comment]
@fcakyon thank you.
please how to get any information of detected image, like count of detected classes or its name

**Comment by fcakyon** [comment]
@totoadel you can get these info from the `object_prediction_list`;

```python
result = get_sliced_prediction(
    "demo_data/small-vehicles1.jpeg",
    detection_model,
    slice_height = 256,
    slice_width = 256,
    overlap_height_ratio = 0.2,
    overlap_width_ratio = 0.2
)

result .object_prediction_list[0]

>> ObjectPrediction<
          bbox: BoundingBox: <(447, 308, 496, 342), w: 49, h: 34>,
          mask: None,
          score: PredictionScore: <value: 0.91552734375>,
          category: Category: <id: 2, name: car>>

```

**Comment by fcakyon** [comment]
Check demo notebooks for more detail: https://github.com/obss/sahi/blob/main/demo/inference_for_yolov5.ipynb
